templates/footer.html

```diff
@@ -0,0 +1,3 @@
+<div class="docs-rs-footer">
+    Who wants some content?
```

Comment:
```suggestion
    <a href="https://foundation.rust-lang.org/policies/privacy-policy/#docs.rs">Privacy</a>
```

Reply:
It can't be the only thing we will put in there, right? Do we also want to add something like "docs.rs is owned by the Rust foundation" or something along the line?

Reply:
I don't think we need to? I would only put things there that are useful things people might want to click, I don't think we should add links just to take up space.

Reply:
Ok!